g11n-ws/vip-common/src/main/java/com/vmware/vip/common/constants/ValidationMsg.java

```diff
@@ -22,7 +22,7 @@ public final class ValidationMsg {
 	public final static String COMBINE_NOT_VALIDE = "Incorrect combine(only 1 or 2)";
 	public final static String SCOPE_NOT_VALIDE = "Incorrect scope(allows letter, comma)";
 	public final static String COLLECTSOURCE_NOT_VALIDE = "Incorrect collectsource(only allows true, false)";
-	public final static String COLLECTSOURCE_NOT_VALIDE_L10N = "Incorrect collectsource(only allows true)";
+	public final static String COLLECTSOURCE_NOT_VALIDE_L10N = "Incorrect collectSource(only allows true, please use translation-product-component-api to confirm received source status)";
```

Comment:
Does it need to change this message?
The source isn't collected successfully, so don't need to tell users how to verify.

Reply:
Yes, because the requirement